README.md

```diff
@@ -1,41 +1,67 @@
 # redactyl.js
 Redact sensitive information from JSON for logging (Node.js)
 
+<p>
+<a href="https://www.npmjs.com/package/redactyl.js"><img alt="npm" src="https://img.shields.io/npm/v/redactyl.js"></a>
+<a href="https://www.npmjs.com/package/redactyl.js"><img alt="npm" src="https://img.shields.io/npm/dw/redactyl.js"></a>
+</p>
+
 ## Installation
 ```sh
 npm install -S redactyl.js
 ```
 
-## Quick Start
+## Usage
 Instantiate a new instance of Redactyl, specifying the properties you wish to redact
+
 ```javascript
 const Redactyl = require('redactyl.js');
 
 let redactyl = new Redactyl({
-  'properties': [ 'apiKey', 'password', 'phone' ]
+  // Required
+  'properties': [ 'apiKey', 'password', 'phone' ],
+
+  // Optional
+  'text': '[REDACTED]',
+  'replacer': myCustomReplacerFunction
 });
 ```
+
+Optional - provide your own replacer function:
```

Comment:
I would add a mention or a link to eg. MDN about this. It was obvious looking at the code that it is the replacer for `JSON.stringify` but here I assumed that it's meant for custom way of redacting values.